src/D2L.Bmx/Program.cs

```diff
@@ -185,7 +193,8 @@
 		duration: context.ParseResult.GetValueForOption( durationOption ),
 		nonInteractive: context.ParseResult.GetValueForOption( nonInteractiveOption ),
 		output: context.ParseResult.GetValueForOption( outputOption ),
-		profile: context.ParseResult.GetValueForOption( profileOption )
+		profile: context.ParseResult.GetValueForOption( profileOption ),
+		cacheAwsCredentials: context.ParseResult.GetValueForOption( cacheAwsCredentialsOption )
```

Comment:
Renamed this flag outside the AWS namespace to make it clear exactly what we are caching when this is set. The CLI option still remains ``--cache``